Core/HLE/sceIo.cpp

```diff
@@ -555,7 +555,7 @@ static VFSFileSystem *flash0System = nullptr;
 
 static void __IoManagerThread() {
 	setCurrentThreadName("IO");
-	while (ioManagerThreadEnabled && coreState != CORE_ERROR && coreState != CORE_POWERDOWN) {
+	while (ioManagerThreadEnabled && coreState != CORE_BOOT_ERROR && coreState != CORE_RUNTIME_ERROR && coreState != CORE_POWERDOWN) {
```

Comment:
Hm, maybe should replace this with some `Core_IsFoo()` at this point...

-[Unknown]

Reply:
Yeah, but can't decide what to name it..